cernopendata/modules/theme/assets/semantic-ui/js/search/components/Facets.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * -*- coding: utf-8 -*-
+ *
+ * This file is part of CERN Open Data Portal.
+ * Copyright (C) 2021 CERN.
+ *
+ * CERN Open Data Portal is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * CERN Open Data Portal is distributed in the hope that it will be
+ * useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with CERN Open Data Portal; if not, write to the
+ * Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+ * MA 02111-1307, USA.
+ *
+ * In applying this license, CERN does not
+ * waive the privileges and immunities granted to it by virtue of its status
+ * as an Intergovernmental Organization or submit itself to any jurisdiction.
+ */
+
+import React from "react";
+import { BucketAggregation, Toggle } from "react-searchkit";
+
+
+const CODFacets = ({ aggs }) => {
+  return (
+    <>
+      <Toggle
+        title="Availability"
+        label="View ondemand datasets"
+        filterValue={["ondemand", "true"]}
+      />
+      {aggs.map((agg) => {
+        return (
+          <div className="ui accordion facet-margin" key={agg.title}>
+            <BucketAggregation title={agg.title} agg={agg.agg} />
+          </div>
+        );
+      })}
```

Comment:
Why it is needed to wrap it with these extra css classes? I've tested
```
  {aggs.map((agg) => (
    <BucketAggregation key={agg.title} title={agg.title} agg={agg.agg} />
  ))}
  ```
  and it works exactly the same for me, margin between the facets is identical. Maybe I'm missing something?

Reply:
I took the component from RDM and now following the native component. Removed the extra div. Thanks for the catch.